index.bs

```diff
@@ -1450,6 +1451,47 @@ that are returned to the caller when a new credential is created, or a new asser
         [=extension identifier=] → [=client extension output=] entries produced by the extension's
         [=client extension processing=].
 
+    :   {{PublicKeyCredential/toJSON()}}
+    ::  This operation returns {{RegistrationResponseJSON}} or {{AuthenticationResponseJSON}}, a JSON-serialized representation of the instance of
+        {{PublicKeyCredential}} suitable for submission to a [=[RP]=] server as a payload with `application/json` mimetype. The [=client=] is in
+        charge of encoding any {{ArrayBuffer}} values using [=base64url encoding=] to generate strings compatible with JSON syntax. {{ArrayBuffer}}
```

Comment:
You need to specify if it's padded or unpadded since there are two variants of base64url.

Reply:
This links to the existing ["Base64url encoding"](https://www.w3.org/TR/webauthn-2/#sctn-dependencies) entry in the Dependencies section of the spec. It states the following:

> The term Base64url Encoding refers to the base64 encoding using the URL- and filename-safe character set defined in Section 5 of [[RFC4648]](https://www.w3.org/TR/webauthn-2/#biblio-rfc4648), with all trailing '=' characters omitted (as permitted by Section 3.2) and without the inclusion of any line breaks, whitespace, or other additional characters.

Might this entry instead be updated to explicitly state "unpadded", rather than implying it with "with all trailing '=' characters omitted'"? Or is padded/unpadded not related to the trailing =?

Reply:
The trailing = represents the padding, yes. So this means it's base64url with no padding vs base64url with padding (which has the ==). But even the inline references to "base64url" generally the accepted default is "padding is default" meaning that if I was to read this, I would interpret this as base64url with padding (unless I checked the references). Many developers and readers will not follow this reference and will assume they know what base64url encoding is here! It's much better to be explicit in the text (base64url without padding) and that links to the section you mention.

Reply:
I disagree with @Firstyear here, the reference is fine as is. Defined terms exist to enhance precision, and readers of any specification should be prepared to look up what the terms mean in that context - especially when uses of the terms are hyperlinked to their definitions. And even if we set that aside, WebAuthn already consistently uses base64url _without_ padding for everything else.

Reply:
I'll just leave things as-is here and lean on the definition to continue communicating that Base64URL is unpadded.